docs/developer-docs/multi-chain/bitcoin/overview.mdx

```diff
@@ -31,13 +29,34 @@ The diagram below shows the flow of interacting with the Bitcoin network from a
   />
 </div>
 
-ICP also implements a smart contract that uses these two features to provide a **ckBTC** token (chain-key BTC) that is backed 1:1 by BTC.
-Using the ckBTC token is easier and more convenient compared to using the two features directly.
-The main value of ckBTC is that it can be transferred with the low finality time of ICP (within seconds) and for a fraction of the cost of a bitcoin transfer on the Bitcoin network.
+## How it works
+
+At the protocol-level, both the Bitcoin adapter and chain-key ECDSA features expose an API on the [management canister](/docs/current/developer-docs/smart-contracts/advanced-features/management-canister).
+
+When a canister wants to make a Bitcoin transaction, a request is sent to the Bitcoin subnet that hosts a Bitcoin canister that contains ICP's on-chain Bitcoin state. The request is sent to the Bitcoin network through the Bitcoin adapter and is processed by Bitcoin asynchronously.
+
+## ckBTC
+
+ICP implements a smart contract that uses threshold ECDSA and chain-key cryptography to provide a [**ckBTC**](/docs/current/developer-docs/multi-chain/bitcoin/ckbtc/overview) token (chain-key BTC) that is backed 1:1 by BTC.
+
+The primary value of chain-key tokens, such as ckBTC, is that it can be transferred with the low finality time (within seconds) and for a fraction of the cost of a transaction on the token's native network (in this instance, a BTC transaction on Bitcoin).
+
+## Use-cases and sample apps
+
+Developers that need more advanced interaction with the Bitcoin network need to use the lower-level threshold ECDSA and Bitcoin APIs. An example of such an advanced use case would be working with Bitcoin inscriptions.
+
+The threshold ECDSA API provides a way for smart contracts to sign transactions with their secret keys. The Bitcoin API allows smart contracts to read the Bitcoin state (UTXOs and balances) and send signed transactions.
```

Comment:
The first sentence is repeated here but has been mentioned earlier in the threshold ecdsa feature part.